### PR TITLE
[TensorRT Docs] Extending engine embedding docs

### PR DESCRIPTION
**Pull request by gedoensmax**
Adding to the docs a link on how to embed an externally compiled engine and also mention that in case of an embedded engine the builder resource is not needed.

### CODE REVIEW

**Comment by gedoensmax** [comment]
@chilo-ms

File: docs/execution-providers/TensorRT-ExecutionProvider.md [hunk 1]
```diff
@@ -499,6 +499,8 @@ $./onnxruntime_perf_test -e tensorrt -r 1 /model_database/transformer_model/mode
...
 * The default name of an embedded engine model will have `_ctx.onnx` appended to the end. Users can specify `trt_ep_context_file_path=my_ep_context_model.onnx` to overwrite this default name.
+* If an embedded engine is used the library **`nvinfer_builder_resource` of TensorRT is not required**, which is by far the largest library. This enables the case of shipping a minimal set of libraries in the case that a fixed set of models is used which are packaged as precompield engine.
+* Besides everything that embedded engines enable to accelerate the load time, they also **enable pacakging an externally compiled engine** using e.g. `trtexec`. A [python script](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/tensorrt/gen_trt_engine_wrapper_onnx_model.py) that is capable of packaging such a precompiled engine into an ONNX file is included in the python tools.
```
Comment (jywu-msft) [inline docs/execution-providers/TensorRT-ExecutionProvider.md:503]: typo: packaging

**Comment by jywu-msft** [comment]
thanks!

**Comment by chilo-ms** [comment]
Thanks!
BTW, i had a typo in my previous PR, could you help make a fix ("bask" -> "bash") for me.
image

**Comment by gedoensmax** [comment]
Sure @chilo-ms, if I get a quick merge in return ;)

**Review by chilo-ms — APPROVED** [review]

**Pull request merged**